test/platform/test-platform-identify.ts

```diff
@@ -0,0 +1,131 @@
+/// <reference types="@google/local-home-sdk" />
+import test from 'ava';
+import {
+  MockLocalHomePlatform,
+  ERROR_UNDEFINED_IDENTIFYHANDLER,
+  ERROR_UNDEFINED_APP,
+  ERROR_LISTEN_NOT_CALLED,
+  ERROR_UNDEFINED_VERIFICATIONID,
+} from '../../src/platform/mock-local-home-platform';
+import {AppStub} from '../../src/platform/smart-home-app';
+import {injectSmarthomeStubs} from '../../src';
+
+const DISCOVERY_BUFFER = Buffer.from('discovery buffer 123');
+const APP_VERSION = '0.0.1';
+const DEVICE_ID = 'device-id-123';
+
+test.before(t => {
+  injectSmarthomeStubs();
+});
+
+test('trigger-identify-with-undefined-app-throws', async t => {
+  const mockLocalHomePlatform = MockLocalHomePlatform.getInstance(true);
+  t.is(
+    (
+      await t.throwsAsync<Error>(
+        mockLocalHomePlatform.triggerIdentify(DISCOVERY_BUFFER)
```

Comment:
Can you explain why this may be an error?

Reply:
Added comments explaining all tests.  This test throws because there was no `App` ever created and attached to the Local Home Platform.  There would be no handler to pass a buffer to.